samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java

```diff
@@ -68,7 +70,34 @@
 
     @Override
     public void start(Stage stage) {
-        MapView view = new MapView();
+    	TileRetriever osm=new CachedOsmTileRetriever() {
+    		 public String buildImageUrlString(int zoom, long i, long j) {
+    			 return "http://tile.openstreetmap.org/" +zoom+"/"+ i + "/" + j + ".png";
+    			    
+    		 }
+    		 public String copyright() {
+    			 return "Map © OpenStreetMap contributors CC-BY-SA -- Please switch to another tile provider ";
```

Comment:
If I understand the issue, we shouldn't be using OpenStreetMap tile API in samples. However, saying "Please switch to another tile provider" is vague. Should we instead change this to:

> Map data © OpenStreetMap contributors, CC-BY-SA. Imagery © OpenStreetMap, for demo only.

Reply:
Done . AFAIK in the sample app the main issue was that there was no licence attribution